README.md

```diff
@@ -1,2 +1,44 @@
 # bigquery-maptiles
-A service that manages the conversion of M-Lab BigQuery results into static world map tiles
+
+Build container:
+
+    docker build -t bigquery-maptiles .
```

Comment:
In Markdown, indenting like this creates a blockquote.  For commands to literally enter into a command prompt, [code blocks](https://help.github.com/en/articles/creating-and-highlighting-code-blocks#fenced-code-blocks) are more appropriate.  Should look like:
````md
```sh
docker build -t bigquery-maptiles .
```
````

Reply:
Done.